code/datums/mutations/touch.dm

```diff
@@ -32,6 +32,11 @@
 		return
 	if(iscarbon(target))
 		var/mob/living/carbon/C = target
+
+		if(C.stat == DEAD && !user.combat_mode)
+			defib(C, user)
+			return
+
 		if(C.electrocute_act(15, user, 1, SHOCK_NOGLOVES | SHOCK_NOSTUN))//doesnt stun. never let this stun
```

Comment:
say, since you're touching shock touch code anyway, could you do me a favor? when I added SHOCK_NOGLOVES to the flags of this electrocute_act(), I forgot to add it to the `L.electrocute_act(15, user, 1, SHOCK_NOSTUN)` used for living noncarbons just a couple of lines below this. if you could fix that for me, I'd really appreciate it (and it would earn you a bit of GBP by making this PR technically count as a fix).

Reply:
this is a feature pr, not a bug fix pr